bundles/org.openhab.binding.hue/doc/readme_v2.md

```diff
@@ -0,0 +1,237 @@
+The `event_id` can have the following values:
+
+| Event                | Value |
+|----------------------|-------|
+| INITIAL_PRESS        | 0     |
+| REPEAT               | 1     |
+| SHORT_RELEASE        | 2     |
+| LONG_RELEASE         | 3     |
+| DOUBLE_SHORT_RELEASE | 4     |
+
+So (for example) the channel value `1002` ((1 * 1000) + 2) means that the second button in the device had a short release event.
+
+The `rotary-steps` channel is a trigger channel.
+When the dial is turned, the channel receives a number corresponding to the number of steps of the last movement of a rotary dial.
+A positive number means the dial was rotated clock-wise, whereas a negative number means it was rotated counter-clockwise.
+
+### Channels for Rooms and Zones
+
+Room and Zone things allow you to control the lights in a given zone or room.
+They support the following channels:
+
+| Channel ID          | Item Type          | Description                                                                       |
+|---------------------|--------------------|-----------------------------------------------------------------------------------|
+| brightness          | Dimmer             | Supports adjusting the brightness or switching the lights on and off.             |
+| switch              | Switch             | Supports switching the lights on and off.                                         |
+| scene<sup>1)</sup>  | String             | Setting the string to a valid scene friendly name activates the respective scene. |
+| dynamics            | Number:Time        | The duration of dynamic transitions between light or scene states.                |
+| alert<sup>1)</sup>  | String             | This channel allows setting an alert on the lights e.g. flashing them.            |
+
+<sup>1)</sup> The scene and alert channels are optional.
+If the respective room or zone has no scenes or alerts associated with it, the respective channel will not be shown.
+
+### The `dynamics` Channel
+
+Some channels support dynamic transitions between light states.
+A dynamic transition is where, instead of the light state changing immediately to its new target value, it changes gradually to the new value over a period of time.
+
+If a thing supports dynamic transitions, then it will have a `dynamics` channel.
+This is a numeric channel where you can set the time delay for the transition in milliseconds.
+When you set a value for the `dynamics` channel (e.g. 2000 milli seconds) and then quickly issue another command (e.g. brightness 100%), the second command will be executed gradually over the period of milliseconds given by the `dynamics` channel value.
+When the `dynamics` channel value is changed, it triggers a time window of ten seconds during which the value is active.
+If the second command is sent within the active time window it will be executed gradually according to the `dynamics` channel value.
```

Comment:
```suggestion
If the second command is sent within the active time window, it will be executed gradually according to the `dynamics` channel value.
```